change/@ni-nimble-angular-6221e053-9114-4da5-a449-c8e19b91db59.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "major",
```

Comment:
Usual reminder to think through communication plan and who does client updates (if needed) for the breaking change.

Reply:
Not applicable anymore since this is no longer a breaking change.